redata/models/data_source.py

```diff
@@ -91,7 +91,7 @@ def get_db_object(self):
             )
             return BigQuery(self, db, schema=self.schemas)
 
-        db = create_engine(db_url)
+        db = create_engine(db_url, connect_args={'sslmode': 'allow'})
```

Comment:
There is one problem with pymssql (used for sql server) it crashes when this parameter is passed.
Would you be able to modify how crate_engine is called so that for sql server sslmode is not passed?

`  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/engine/default.py", line 493, in connect
    return self.dbapi.connect(*cargs, **cparams)
  File "src/pymssql.pyx", line 582, in pymssql.connect
TypeError: connect() got an unexpected keyword argument 'sslmode'`

Reply:
Okay I will work on a fix

Reply:
I have a fix but will be difficult to implement because of how you implemented the function below in `redata/db_operations.py`

```
def get_metrics_connection():
    db_string = settings.METRICS_DB_URL
    db = create_engine(db_string)
    return db
```

The function does not take into consideration the source type of the database. It makes the assumption that all the databases will be okay with the way the `create_engine` function has been used.

Reply:
I have a fix now and I think it should be okay. I will be updated the PR soon

Reply:
Great! :) Also just to make sure it's clear, `get_metrics_connection` function above is called only for internal database (which means also only for postgresql) so this doesn't need to be changed.

Reply:
Yeah, you are right. I figured out that late that was why I deleted my comment.
I just pushed a fix. You can have a look and test.

Reply:
Looked good and works. 👍  Merging